scripts/maintenance/migrate_project/src/models.py

```diff
@@ -0,0 +1,86 @@
+import json
+from pathlib import Path
+from typing import Optional
+from uuid import UUID
+
+from pydantic import BaseModel, Field
+from sqlalchemy import desc
+
+
+class DBConfig(BaseModel):
+    address: str
+    user: str
+    password: str
+    database: str
+
+
+class S3Config(BaseModel):
+    endpoint: Optional[str] = None
```

Comment:
add provider here to maybe enable interoperability with all providers?

Reply:
Will do in future if this is required.